### PR TITLE
Add download policies constant

### PR DESCRIPTION
**Pull request by bmbouter**
https://pulp.plan.io/issues/4243
re #4243

### CODE REVIEW

File: pulp_smash/pulp3/constants.py [hunk 1]
```diff
@@ -20,6 +20,8 @@
 
 DISTRIBUTION_PATH = urljoin(BASE_PATH, 'distributions/')
 
+DOWNLOAD_POLICIES = ['immediate', 'on_demand', 'streamed']
```
Comment (kersommoura) [inline pulp_smash/pulp3/constants.py]: Did we rename `cahe-olny`?
```python
DOWNLOAD_POLICIES = ['cache_only', 'immediate', 'on_demand']
```
Constant on [Pulp file](https://github.com/pulp/pulp_file/blob/c7a90288e96d45faac191809cc9be8cca94a2117/pulp_file/tests/functional/constants.py#L12)

Comment (bmbouter) [reply pulp_smash/pulp3/constants.py]: I did so in another PR but we can do this in any order/way you recommend.

https://github.com/pulp/pulp_file/pull/140/files#diff-2d4578fd30e72ae31e7baef653596f7fL12

**Comment by bmbouter** [comment]
@pulp/QE I'd like to get this merged and released. I just realized it's failing these tests here: https://github.com/ATIX-AG/pulp_gem/pull/14

**Comment by bmbouter** [comment]
Oh this is merged already, my page wasn't refreshed. When can this be released to pypi?

**Comment by rochacbruno** [comment]
@bmbouter released

File: pulp_smash/pulp3/constants.py (hunk 1 above)
Comment (rochacbruno) [reply pulp_smash/pulp3/constants.py]: https://github.com/PulpQE/pulp-smash/issues/1157

**Comment by bmbouter** [comment]
@rochacbruno Thank you!

**Pull request merged**